src/common/fetch.ts

```diff
@@ -107,9 +107,16 @@ export const fetchLists = async (supabase: Supabase): Promise<Schema["lists"][]>
 };
 
 export const fetchComment = async (supabase: Supabase): Promise<string> => {
-    const { data, error } = await supabase.from("website_data").select().eq("name", "lists_text");
+    const { data, error } = await supabase
+        .from("website_data")
+        .select()
+        .eq("name", "lists_text")
+        .limit(1)
+        .single();
```

Comment:
Ah great, so it will actually recognise the error now

Reply:
Btw [VFB-57](https://softwiretech.atlassian.net/browse/VFB-57) is about the fact that nothing useful happens with the subsequent DatabaseError

Reply:
Thanks for the info!